catalog/spatial/geocoding/spatial-geocoding-extract/pom.xml

```diff
@@ -73,6 +92,10 @@
                     <instructions>
                         <Bundle-SymbolicName>${project.artifactId}</Bundle-SymbolicName>
                         <Embed-Dependency>
+                            platform-utils,
+                            commons-validator,
```

Comment:
✏️ We shouldn't be embedding commons-validator, commons-beanutils or commons-digester. They are bundles. Instead we should make sure it is started with this feature.